app/views/stocks/index.html.erb

```diff
@@ -8,7 +8,12 @@
       <th>Name</th>
       <th>Info</th>
       <th>Unit</th>
+      <th>City</th>
+      <th>Street</th>
+      <th colspan="3"></th>
```

Comment:
please check if the number <th> is equal to <td>'s... otherwise our JS tables are not working. Also remove colspan please